examples/armor.toml

```diff
@@ -0,0 +1,8 @@
+asset_type        = "armor"
```

Comment:
I'm unsure about an `asset_type` of `"armor"`. Do we want them to get that fine-grained? I don't necessarily want to hold this PR up on this one thing, but I think this is the point where we should start this discussion, as you brought up in #3. What is our full `asset_type` list? Do we have `asset_type`s like `"armor"`, or do we use some form of "tagging" to support an [entity component system](https://en.wikipedia.org/wiki/Entity_component_system)? What would that even look like?

Random brain dump possibility:

```toml
asset_type = "item"  # Asset == Entity
asset_components = [
    "wearable"
]
```

Another example might be:

```toml
asset_type = "item"
asset_components = [
    "wearable",
    "container"
]
short_description = "a hoodie with pockets"
```

I dunno. I'm rambling. I will probably copy this stuff over to #3 to keep the discussion going there.

Reply:
I think having a tag system is a fine idea. I'll discuss it more on #3.